media/css/firefox/whatsnew/whatsnew-rocket.scss

```diff
@@ -0,0 +1,83 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
```

Comment:
Nit: this page isn't using our new standard margins? `@import '../../hubs/sections';`